packages/types/lib/Entity.ts

```diff
@@ -56,30 +35,31 @@ export interface EntityMeta extends EntityBase {
  * @example
  * const testCourse = new Entity('content/courses/test-course');
  */
-export class Entity implements EntityMeta {
-  public readonly title: string;
-  public readonly slug: string;
-  public readonly type: EntityType;
-  public readonly tags = new Array<string>();
-  public readonly previous?: string;
-  private _children?: Array<EntityMeta>;
-  readonly [key: string]: unknown;
+// export class Entity implements EntityMeta {
+//   public readonly title: string;
+//   public readonly slug: string;
+//   public readonly type: EntityType;
+//   public readonly tags = new Array<string>();
+//   public readonly previous?: string;
+//   private _children?: Array<EntityMeta>;
+//   readonly [key: string]: unknown;
 
-  constructor(public readonly address: string) {
-    // Assign the slug based on the directory name from the given address
-    this.slug = path.dirname(address);
+//   constructor(public readonly address: string) {
+//     // Assign the slug based on the directory name from the given address
+//     this.slug = path.dirname(address);
 
-    // Retrieve entity metadata from disk
-    const { type, title, tags, previous, ...dynamic } = getEntityMetaSync(address);
-    this.type = type;
-    this.title = title;
-    this.tags = tags;
-    this.previous = previous;
-    Object.assign(this, dynamic);
-  }
+//     // Retrieve entity metadata from disk
+//     const { type, title, tags, previous, ...dynamic } =
+//       getEntityMetaSync(address);
+//     this.type = type;
+//     this.title = title;
+//     this.tags = tags;
+//     this.previous = previous;
+//     Object.assign(this, dynamic);
+//   }
 
-  public get children(): Array<EntityMeta> {
-    // Retrieve child entities from disk if not already cached
-    return this._children || (this._children = getChildrenSync(this.address));
-  }
-}
+//   public get children(): Array<EntityMeta> {
+//     // Retrieve child entities from disk if not already cached
+//     return this._children || (this._children = getChildrenSync(this.address));
+//   }
+// }
```

Comment:
Should all of this stay commented out?

Reply:
In this case yes, I do want those comments there. It's an example implementation but not something that will end up in final code and it will be forgotten if I isolate it in a branch. Once we move some functions over from LMS it can be finished but for now its not being used and I don't want to delete it or isolate it in a branch